jobs/build-arch.Jenkinsfile

```diff
@@ -92,9 +94,24 @@ def cosa_memory_request_mb = 2.5 * 1024 as Integer
 pod = pod.replace("COREOS_ASSEMBLER_CPU_REQUEST", "1")
 pod = pod.replace("COREOS_ASSEMBLER_CPU_LIMIT", "1")
 
+// We recently did some golang migration [1] and happened to add the
+// `cosa remote-session` pieces [2] after that migration. This makes
+// it not easy to backport to ther branches. For a period of time
+// let's use the latest COSA for our COSA pod for multi-arch builds
+// instead of the versioned COSA. We will still use the versioned COSA
+// on the remote so the actual building will happened with the correctly
+// versioned COSA (substituted below and passed as an argument to
+// `cosa remote-session create --image=`).
+// [1] https://github.com/coreos/coreos-assembler/pull/2919
+// [2] https://github.com/coreos/coreos-assembler/pull/2979
+def cosa_pod_image = params.COREOS_ASSEMBLER_IMAGE
+if (cosa_pod_image =~ '^coreos-assembler:rhcos-4.(6|7|8|9|10|11)$') {
```

Comment:
Can we make that more generic to not have to edit it when 4.12 gets out?

Reply:
We won't - 4.12 supports `cosa remote-session` so we won't need to do any special handling there. When this list is exhausted we'll remove this code.

Reply:
nice